applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -174,14 +188,52 @@ def Initialize(self):
 
         self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DYNAMIC_TAU, self.settings["dynamic_tau"].GetDouble())
 
-        KratosMultiphysics.Logger.PrintInfo("NavierStokesEmbeddedMonolithicSolver", "Solver initialization finished.")
+        # For the primitive Ausas formulation, set the find nodal neighbours process
+        # Recall that the Ausas condition requires the nodal neighbouts.
+        if (self.settings["solver_type"].GetString() == "EmbeddedAusas"):
+            number_of_avg_elems = 10
+            number_of_avg_nodes = 10
+            self.find_nodal_neighbours_process = KratosMultiphysics.FindNodalNeighboursProcess(self.GetComputingModelPart(),
+                                                                                               number_of_avg_elems,
+                                                                                               number_of_avg_nodes)
+
+        # If required, create the FM-ALE utility
```

Comment:
I think it would be cleaner to have a solver deriving from this which does the fm_ale stuff

you are mixing up things which I think makes it harder to understand / follow the code

Reply:
This is what we had before this PR and we (I'm speaking in behalf of @jcotela)  decided to remove it. The point is that such fm_ale solver has barely difference with the "common" one. We both think that it doesn't pay the price to have an alternative one (the FM-ALE util is deactivated by default). Indeed, any moving boundaries problem should be accompanied by the FM-ALE utility,

Reply:
ok